en/api/nuxt.md

```diff
@@ -26,6 +26,7 @@ const config = require('./nuxt.config.js')
 
 // Create a new Nuxt instance
 const nuxt = new Nuxt(config)
+await nuxt.ready()
```

Comment:
await is not possible on top level code. this requirement has been also removed until nuxt3

Reply:
I see. Thanks.